stabilization-guide.md

```diff
@@ -3,6 +3,41 @@ layout: default
 title: So you want to stabilize a feature?
 ---
 
+## Request for stabilization
+
+Once an unstable feature has been well-tested with no outstanding concern, anyone may push for its stabilization. It involves the following steps.
+
+### Documentation PRs
+
+Prepare PRs to update the documentations involving this new feature. You will likely need to submit PRs to [The Reference], [The Book] and [Rust by Example]. See the [Updating documentation](#updating-documentation) at the end of this page for detailed requirements.
+
+Maintainers of these repositories will keep these PRs open until the whole stabilization process has completed. Meanwhile, we could proceed to the next step.
```

Comment:
You might want to say `can` rather than `could` because it implies that you shouldn't have to wait around for these PRs to land. If you want the language to be even stronger with this using `should` is another good choice! :D